content/blog/2022/2022-04-08-interactions.md

```diff
@@ -0,0 +1,37 @@
++++
+author = "Noah Novšak"
+date = "2022-04-08"
+draft = true
+title = "Predictive Modelling with Attribute Interactions"
+type = "blog"
+thumbImage = "/blog_img/2022/image-20220408145950556.png"
+frontPageImage = "/blog_img/2022/image-20220408145950556.png"
+blog = ["orange"]
+shortExcerpt = "Interactions widget in Orange 3"
+longExcerpt = "The Interactions widget has been added back to Orange 3. Illustrating how to use attribute interactions to improve predictive models."
+x2images = true  # true if using retina screenshots, else false
++++
+
+The *Interactions* widget is one of the newest additions to Orange. Previously only available in Orange 2, it has been rewritten and is accessible as a prototype add-on. This way, one need not go through the trouble of [compiling older versions](/blog/2022/2022-01-10-orange2/) anymore.
```

Comment:
"As a prototype add-on" sounds more like this is some kind of special add-on that is still a prototype. I propose "in the Prototype add-on".